include/deal.II/particles/particle_handler.h

```diff
@@ -660,6 +660,16 @@ namespace Particles
     unsigned int
     n_properties_per_particle() const;
 
+    /**
+     * Return the maximal local index (in MPI-local index space) returned by
+     * ParticleAccessor::get_local_index(). This number can be larger than
+     * locally_owned_particle_ids().n_elements() in case
+     * exchange_ghost_particles() has been called to populate the particles
+     * also on ghost cells.
```

Comment:
This is not the only case where it can be larger. It can also be larger if particles have been removed and empty slots are kept by the property pool. Maybe simplify to:
```suggestion
     * ParticleAccessor::get_local_index(). This number can be larger than
     * locally_owned_particle_ids().n_elements(), because local indices
     * can contain gaps in the range 0 to get_max_local_particle_index().
```
Or something similar (saying that local indices do not have to be consecutive numbers, but not specifying why, because that can change).